opensciencegrid/osdf-collector/etc/condor/config.d/50-osdf-collector.conf

```
@@ -0,0 +1,41 @@
+
+###
+# Configurations for the OSDF collector
+###
+
+# We only need the collector in this container
+DAEMON_LIST = COLLECTOR
+
+# This is the name in the collector ad sent to collector.opensciencegrid.org
+COLLECTOR_NAME=OSDF-Collector
+
+# As of the time of writing, 3DES is still in the default security parameters.
+SEC_DEFAULT_CRYPTO_METHODS = AES
+
+# Enable strong security
+use security:strong
+
+# Cert and key in Kubernetes-style formats
+AUTH_SSL_SERVER_CERTFILE = /etc/condor/pki/tls.crt
+AUTH_SSL_SERVER_KEYFILE  = /etc/condor/pki/tls.key
+
+# Authorization mapfile.
+CERTIFICATE_MAPFILE=/etc/condor/mapfile
+
+# The mapfile should limit the mapped OSDF caches to *@osg-htc.org
+UID_DOMAIN = osg-htc.org
+ALLOW_DAEMON=*@osg-htc.org
+
+# Limit the kinds of ads we are permitting.
+COLLECTOR_REQUIREMENTS = ( TARGET.MyType == "OSDFServer" || TARGET.MyType == "OSDFNamespace" || TARGET.MyType == "Collector" )
+
+# Forward OSDF ads to the CE collector
+CONDOR_VIEW_HOST=collector.opensciencegrid.org:9619
+UPDATE_VIEW_COLLECTOR_WITH_TCP=True
+
+# Not strictly necessary - but convenient for when you exec into the container:
+COLLECTOR_HOST=localhost
+CONDOR_HOST=localhost
+
+# Improve default logging to match our syslog scripts
+ALL_DEBUG=D_CAT,D_SUB_SECOND,D_PID
```

```suggestion
ALL_DEBUG=D_CAT,D_PID
```
The regexps in rsyslog.conf do not appear to match sub-second times.